sw/device/lib/dif/meson.build

```diff
@@ -69,3 +69,17 @@ sw_lib_dif_i2c = declare_dependency(
     ],
   )
 )
+
+# OTBN DIF library
+sw_lib_dif_otbn = declare_dependency(
+  link_with: static_library(
+    'sw_lib_dif_otbn',
+    sources: [
+      hw_ip_otbn_reg_h,
```

Comment:
These are defined in the top-level `meson.build` - adding a line there will solve your issue currently seen in CI.

Reply:
It's coming in as part of another PR, don't worry about build failures at this point.